### ISSUE TITLE
Can't install

### ISSUE THREAD

**Issue by djackson265** [owncloud/core#29088]
### Steps to reproduce
I just set up a brand new LAMP stack and Owncloud installation on a fresh Debian 9 installation (set up only as SSH server in the Linux installation, no desktop, no standards tools, nothing but SSH) following the instructions on these two web pages: 

LAMP stack: http://manjaro.site/install-lamp-server-debian-9/ 
Owncloud: http://manjaro.site/install-owncloud-10-debian-9/

Logged in as root, every command was entered by cut and paste, so there couldn't have been a typo, and there were no error messages. When filled out the form to set up Owncloud, again cutting and pasting from a Notepad doc all the account names and passwords used to prevent typos, the web-based setup failed with the following:

Error
Error while trying to create admin user:
Failed to connect to the database: An
exception occurred in driver:
SQLSTATE[HY000][1045] Access denied
for user 'owncloud'@'localhost' (using
password: YES)

I've tried this multiple times from a blank drive with no OS and received the same error every time. 

### Expected behaviour
What should happen is the setup process should create the admin account and login to the database.

### Actual behaviour
Error
Error while trying to create admin user:
Failed to connect to the database: An
exception occurred in driver:
SQLSTATE[HY000][1045] Access denied
for user 'owncloud'@'localhost' (using
password: YES)

### Server configuration
**Operating system**:
Debian 9
**Web server:**
Apache (latest version, downloaded today
**Database:**
MariaDB, latest version
**PHP version:**
Latest version
**ownCloud version:** (see ownCloud admin page)
Latest version
**Updated from an older ownCloud or fresh install:**
Fresh install
**Where did you install ownCloud from:**
wget https://download.owncloud.org/community/owncloud-10.0.2.tar.bz2

**Signing status (ownCloud 9.0 and above):**
I don't know what "signing status" is so I can't answer.
```
Login as admin user into your ownCloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results into https://gist.github.com/ and puth the link here.
```
I can't login as the admin user because it won't complete the setup.

**The content of config/config.php:**
 'datadirectory' => '/var/www/html/owncloud/data',
  'overwrite.cli.url' => 'http://xxx.xxx.xxx.xxx/owncloud',
  'dbtype' => 'mysql',
  'version' => '10.0.2.1',
  'dbname' => 'owncloud',
  'dbhost' => 'localhost',
  'dbtableprefix' => 'oc_',

```
```

**List of activated apps:**
I really don't know what you mean by "activated apps." There is nothing on this server except a barebones Debian LAMP stack and Owncloud.
```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your ownCloud installation folder.
```
Result: Could not open input file: occ

**Are you using external storage, if yes which one:** local/smb/sftp/...
Not using anything yet, as it won't set up.
**Are you using encryption:** yes/no
No
**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
Haven't gotten that far yet
```

### Client configuration
**Browser:**
Chrome
**Operating system:**
Windows 7 Pro
### Logs
#### Web server error log
```
Insert your webserver log here
```
[Sat Sep 23 13:00:44.390846 2017] [mpm_event:notice] [pid 1519:tid 139746317178048] AH00489: Apache/2.4.25 (Debian) configured -- resuming normal operations
[Sat Sep 23 13:00:44.391055 2017] [core:notice] [pid 1519:tid 139746317178048] AH00094: Command line: '/usr/sbin/apache2'
[Sat Sep 23 13:05:04.509923 2017] [mpm_event:notice] [pid 1519:tid 139746317178048] AH00491: caught SIGTERM, shutting down
[Sat Sep 23 13:05:04.569064 2017] [mpm_prefork:notice] [pid 9441] AH00163: Apache/2.4.25 (Debian) configured -- resuming normal operations
[Sat Sep 23 13:05:04.569235 2017] [core:notice] [pid 9441] AH00094: Command line: '/usr/sbin/apache2'
[Sat Sep 23 13:05:04.830096 2017] [mpm_prefork:notice] [pid 9441] AH00169: caught SIGTERM, shutting down
[Sat Sep 23 13:05:04.990337 2017] [mpm_prefork:notice] [pid 9496] AH00163: Apache/2.4.25 (Debian) configured -- resuming normal operations
[Sat Sep 23 13:05:04.990459 2017] [core:notice] [pid 9496] AH00094: Command line: '/usr/sbin/apache2'
[Sat Sep 23 13:11:15.680466 2017] [mpm_prefork:notice] [pid 9496] AH00169: caught SIGTERM, shutting down
[Sat Sep 23 13:11:15.807139 2017] [mpm_prefork:notice] [pid 17044] AH00163: Apache/2.4.25 (Debian) configured -- resuming normal operations
[Sat Sep 23 13:11:15.807245 2017] [core:notice] [pid 17044] AH00094: Command line: '/usr/sbin/apache2'

#### ownCloud log (data/owncloud.log)
```
Insert your ownCloud log here
```
Owncloud log is empty.
#### Browser log
```
Insert your browser log here, this could for example include:
No log to insert

**Comment by phil-davis** [comment]
That process should work fine. You can now try with the recent 10.0.3 also.
This is the place for bug reports. Please first ask for help at https://central.owncloud.org/ and then coe back here if an actual bug is found.

**Comment by djackson265** [comment]
I came there because that's where the link for help on OwnCloud's web site took me, and I realize that process "should work fine," but it doesn't, and I got the same result on multiple attempts. That's why I spent the time describing everything so that someone who knows something about it could try that process and either verify that "it does work fine" or tell me why it "doesn't work fine," traditionally the first step in debugging. I don't think that's an unreasonable expectation given the complexity of Linux and the fact that both Debian and OwnCloud have recently had major updates and LAMP components get updated daily. So thanks for the bum's rush, and have a nice day.

Sent from my android device.

-----Original Message-----

**Comment by phil-davis** [comment]
```
SQLSTATE[HY000][1045] Access denied
for user 'owncloud'@'localhost' (using
password: YES)
```
that message is really about some issue between the way the mySQL or MariaDB databse user is set up and what has been entered for the ownCloud installation settings.

There was also an issue if you click to display the password that you are entering at the installation screen - https://github.com/owncloud/core/issues/26129 and fixed for stable10 by PR https://github.com/owncloud/core/pull/28554 That is a fix in 10.0.3

So try 10.0.3 or try not clicking to display the password as it is entered.

**Comment by djackson265** [comment]
Roger that. Thank you.

Sent from my android device.

-----Original Message-----
